components/Search/Hit.tsx

```diff
@@ -77,32 +93,49 @@ If neither the excerpt or the description is a match (like the match can be for
   a resonsible person) we will fallback to either the excerpt or description, like
   in the static text without any query)
 */
+
 const getSnippetAttribute = (hit: HitProps["hit"]) => {
   // eslint-disable-next-line no-underscore-dangle
   const excerptSnippet = hit._snippetResult?.excerpt as HitAttributeSnippetResult
   // eslint-disable-next-line no-underscore-dangle
   const descriptionSnippet = hit._snippetResult?.description as HitAttributeSnippetResult
+
   if (hit.excerpt && excerptSnippet && excerptSnippet.matchLevel === ("full" || "partial")) {
     return "excerpt"
     /* eslint no-else-return: "error"  */
   } else if (hit.description && descriptionSnippet && descriptionSnippet.matchLevel === ("full" || "partial")) {
     return "description"
   }
+
   return hit.excerpt ? "excerpt" : "description"
 }
 
+const isNewDataProduct = (hit: HitProps["hit"]) => {
+  const currentDate = new Date()
+  const compareDate = new Date(currentDate.setMonth(currentDate.getMonth() - 1))
+  const createdAtDate = new Date(hit.createdAt)
+  return createdAtDate.getTime() >= compareDate.getTime()
+}
+
 export const Hit = ({ hit, hasQuery }: HitProps) => {
   const { id, community = [], tags = [] } = hit
+  const intl = useIntl()
 
   return (
     <StyledLink href={{ pathname: "/assets/[id]", query: { id } }}>
-      {community &&
-        community.map((item) => (
-          <Typography variant="overline" style={{ fontSize: "0.75rem", fontWeight: "medium" }} key={item}>
-            {item}
-          </Typography>
-        ))}
-
+      <TopContainer>
+        {community &&
+          community.map((item) => (
+            <Typography variant="overline" style={{ fontSize: "0.75rem", fontWeight: "medium" }} key={item}>
+              {item}
+            </Typography>
+          ))}
+        {isNewDataProduct(hit) && (
+          <StyledChipNew style={{ fontSize: "0.75rem", fontWeight: "medium" }}>
```

Comment:
Medium is not a known value for font weight, but EDS adds 500 and that's the same as Figma. Should remove this